modules/user_manual/pages/files/public_link_shares.adoc

```diff
@@ -1,49 +1,47 @@
 
-As of ownCloud version 10.0.2 you can xref:files/webgui/sharing.adoc#creating-drop-folders[create Drop Folders], 
-where users can upload files to a central location, but not be able to change any existing ones, nor see other 
-files which already have been uploaded.
+As of ownCloud version 10.0.2 you can xref:files/webgui/sharing.adoc#creating-drop-folders[create Drop Folders], where users can upload files to a central location, but not be able to change any existing ones, nor see other files which already have been uploaded.
 
-[[creating-public-link-shares]]
 == Creating Public Link Shares
 
-To create a public link share, first view the Sharing Panel of the file
-or folder that you want to create a public link share for, and under
-*"Public Links"* click *"Create public link"*.
+To create a public link share, first view the Sharing Panel of the file or folder that you want to create a public link share for, and under *"Public Links"* click *"Create public link"*.
 
 image:public-link/create-public-link.png[Create a public link - step one.]
 
-Then, as with other shares, provide the name in the *"Link Name"*
-field, and fill out the options that suit what you want the link to
-support. You can find details of what each option does below.
+Then, as with other shares, provide the name in the *"Link Name"* field, and fill out the options that suit what you want the link to support. 
+You can find details of what each option does below.
 
 image:public-link/public-link-settings.png[Create a public link - step two.]
 
-Finally, click *"Save"* to complete creation of the share. Now that the share is created, you can:
+Finally, click *"Save"* to complete creation of the share. 
+Now that the share is created, you can:
 
 * Copy the link to the share and give it out
 * Update the share’s settings
 * Share the link via social media and email
 * Delete the public link
 
-[[share-settings-explained]]
 === Share Settings Explained
 
 [cols=",",options="header",]
 |===
 | Setting
 | Description
 
-| Allow editing
-| It also allows renaming, deleting, and moving.
+| Download / View
+| Allows recipients to view or download the public link's contents.
 
-| Show file listing
-| The option allows recipients to list the files in the shared folder. If you don’t give permission to view the files you have xref:files/webgui/sharing.adoc#creating-drop-folders[a drop folder].
+| Download / View / Edit
+| Allows recipients to view, download, edit, and delete the public link's contents.
+
+| Download / View / Upload
+| Allows recipients to view, download and upload the public link's contents.
+
+| Upload only (File Drop)
+| Allows recipients to create xref:files/webgui/sharing.adoc#creating-drop-folders[a drop folder], which can receive files from multiple recipients without revealing the contents of the folder.
```

Comment:
Allows recipients to create - ?
maybe you mean:
Allows users to create...

Reply:
All that was done here was a small variation from the text in the modal dialog. That said, if "users" makes more sense, fine by me.

Reply:
Actually, would "users" or "share recipients" make more sense as well as being technically correct?

Reply:
Personally "share recipients" ( more explicit and less confusion ) looks ok to me. But if technically "users" is better than "share recipients", then I am fine with that too.

Reply:
Thanks, @sharidas. @phil-davis?

Reply:
My point was that "share recipients" are not the ones who create a "drop folder". Ordinary users create a "drop folder". And then recipients (people who are given the link) can do the stuff in the 2nd half of the sentence.

Reply:
Ah, okay. Understood.